### PR TITLE
Feature/us3333 known spouse attending

### PR DESCRIPTION
**Pull request by andycanterbury**

### CODE REVIEW

File: Gateway/crds-angular/Controllers/API/ProfileController.cs [hunk 1]
```diff
@@ -130,6 +135,29 @@ public IHttpActionResult AdminGetProfile(int contactId)
...
+                    Email = s.Email_Address,
+                    LastName = s.Last_Name,
+                    PreferredName = s.Preferred_Name,
+                    RelationshipId = s.Relationship_Id,
+                    ParticipantId = s.Participant_Id,
+                    HighSchoolGraduationYear = s.HighSchoolGraduationYear
+                }).SingleOrDefault();
+                return Ok(spouse);
+            });
+        }
```
Comment (IngageMaddoxT) [inline Gateway/crds-angular/Controllers/API/ProfileController.cs]: error handling needed.

File: crossroads.net/app/go_volunteer/goVolunteer.routes.js [hunk 2]
```diff
@@ -135,6 +136,30 @@
     return deferred.promise;
   }
 
+  function GetSpouse(Profile, $cookies, $q, GoVolunteerService, $stateParams) {
+    var deferred = $q.defer();
+
+    if ($stateParams.page === 'spouse') {
+      var cid = $cookies.get('userId');
```
Comment (IngageMaddoxT) [inline crossroads.net/app/go_volunteer/goVolunteer.routes.js]: cid, what does that mean?

File: crossroads.net/app/go_volunteer/page/spouseName/goVolunteerSpouseName.component.js [hunk 3]
```diff
@@ -16,9 +20,40 @@
...
+
       var vm = this;
 
+      vm.birthdateOpen = false;
+      vm.initDate = new Date(now.getFullYear(), now.getMonth(), now.getDate());
+      vm.maxBirthdate = new Date(now.getFullYear() - 18, now.getMonth(), now.getDate());
+      vm.oneHundredFiftyYearsAgo = new Date(now.getFullYear() - 150, now.getMonth(), now.getDate());
+      vm.openBirthdatePicker = openBirthdatePicker;
+      vm.person = GoVolunteerService.person;
+      vm.phoneFormat = /^\(?(\d{3})\)?[\s.-]?(\d{3})[\s.-]?(\d{4})$/;
```
Comment (IngageMaddoxT) [inline crossroads.net/app/go_volunteer/page/spouseName/goVolunteerSpouseName.component.js]: is there a way to pull this out to a common place?  i bet this same pattern is used elsewhere.

File: crossroads.net/app/go_volunteer/goVolunteer.routes.js (hunk 2 above)
Comment (andycanterbury) [inline crossroads.net/app/go_volunteer/goVolunteer.routes.js]: contact id

File: crossroads.net/app/go_volunteer/page/spouseName/goVolunteerSpouseName.component.js (hunk 3 above)
Comment (andycanterbury) [inline crossroads.net/app/go_volunteer/page/spouseName/goVolunteerSpouseName.component.js]: There may be, but I don't know how.

**Pull request merged**